main.rb

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env ruby
+
+# Create methods that do the following:
+# List all books.
+# List all people.
+# Create a person (teacher or student, not a plain Person).
+# Create a book.
+# Create a rental.
+# List all rentals for a given person id.
+
+require './classroom'
+require './person'
+require './teacher'
+require './student'
+
+def list_books
+  books = [Book.new('Harry Potter', 'JK Rowling'),
+           Book.new('Lord of the Rings', 'JRR Tolkien'),
+           Book.new('The Hobbit', 'JRR Tolkien')]
+
+  books.each do |book|
+    puts book.title
+  end
+end
+
+def list_people
+  people = [Teacher.new('Pysics', 20, 'Harry Maguire', parent_permission: false),
+            Student.new('Pysics', 20, 'Jon Snow', parent_permission: false)]
+
+  people.each do |person|
+    puts person.name
+  end
+end
+
+def create_person(type)
+  case type
+  when 'teacher'
+    Teacher.new('Pysics', 20, 'Jon Snow', parent_permission: false)
+  when 'student'
+    Student.new('Class1', 19, 'Trent', parent_permission: false)
+  end
+end
+
+def create_book
+  Book.new('Harry Potter', 'JK Rowling')
+end
```

Comment:
- [ ] This method is meant to allow the user to create a book. However, you are creating a book by yourself here. Kindly remove the book you created and prompt the user to create a book by entering their input which you can get using gets.chomp then use these inputs to create a book.

```suggestion
def create_book
    print "Title: "
    title = gets.chomp

    print "Author: "
    author = gets.chomp

    @books << Book.new(title, author)
    puts "Book created successfully"
  end
```